src/main/java/SchedularFactory.java

```diff
@@ -0,0 +1,20 @@
+
+public class SchedularFactory {
+
+    public TaskSchedular getTaskSchedular(SchedularType schedular, int queueSize){
+
+        switch (schedular){
+            case FIFO:
+                return new FIFOSchedular(queueSize);
+
+            case LIFO:
+                return new LIFOSchedular(queueSize);
+
+            case PRIORITY_BASED:
+                return new PriorityBasedSchedular(queueSize);
+
+            default:
+                throw new IllegalArgumentException("Invalid Schedular Type !");
+        }
+    }
```

Comment:
Break statements are not required here. As all statements are either return statements or throw statements, all break statements will be rendered unreachable.